packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -69,7 +65,7 @@ const Observation = (props) => {
     ...customProps
   } = props;
 
-  const isValidValue = !!result.value;
+  const isValidValue = result && !!result.value;
```

Comment:
There are a few more places in this file were we will want to valid result has been provided.

Reply:
I don't think in this file? The rest of the calls for `result` in this file are in an `if` that uses `isValidValue`.

Reply:
you right 👍